composer.json

```diff
@@ -13,7 +13,9 @@
     "require": {
         "php": "^5.5.9|^7.0",
         "symfony/framework-bundle": "^2.7|3.*",
-        "symfony-cmf/resource": "^1.0"
+        "symfony/options-resolver": "^2.7|3.*",
+        "symfony-cmf/resource": "dev-admin_metadata",
+        "puli/repository": "@beta"
```

Comment:
I'm not sure we can avoid doing this. The packages minimum-stability is `dev`, so the implicit Puli version is `dev-master`, which is "behind" `@beta`. If we remove the minimum-stability then we still need to specify the unstable `@beta` version.